### PR TITLE
Add plugins.txt example file to understand support core txt format

### PR DESCRIPTION
**Pull request by hayderimran7**
Its unknown what exactly is support core txt format, unless you install the plugin, download zip archive and actually see the plugins txt file in plugin folder which is quite gruesome. Even the support-core plugin wiki is void of the txt format. :/
Added a sample plugins.txt file so users can imitate other plugins they wish to install in same way as mentioned in this example file.

### CODE REVIEW

**Comment by hayderimran7** [comment]
Second commit: added ability to add comments to plugins.txt for convenience.
the comment lines starting with # will be dismissed by plugins.sh

**Comment by hairmare** [comment]
It looks like the script isn't strict about any particular format.

I have a plugins.txt that contains lines with spaces, ie:

```
ant latest
git latest
```

+1 on the comment feature though

**Comment by hayderimran7** [comment]
Yes, the script is tied to a specific format for defining your plugin and their version.  
I would suggest installing the "Support Core Plugin" , after you install and restart, you will see a "support" option on the left panel of your jenkins hompage.  
Click on it, select all options, and download the bundle.  
Open the zip folder, go to 'plugins' folder , you will see three text files.  
Open the 'active.txt' , thats the plugin format that actually works with this script..
thats why I added a PR of 'plugins.txt.sample' so one knows how to declare there plugins.  :)

Just add a full colon to your plugin and version like this:
    `ant:latest`

**Comment by hairmare** [comment]
Now i see the comment calling it a support-core plugin -style txt file :)
Thanks for clarifying that, looks like it really has to be ``:``. I seem to have been rather confused.

**Comment by hayderimran7** [comment]
yes, here is my PR of plugins.txt.sample file for clarity https://github.com/hayderimran7/jenkins-ci.org-docker/commit/ed70162974b6a6d85ad4db52e5fef832ff92f93b
its really hard to know when the author says 'support core plugin format ' like wtf :)
i had to download the plugin, play around a little bit and then came to know how should i declare my plugins ..meh .  
anyway i have made lot of changes to this source code..like added more stuff to init.groovy script, and changing the dockerfile..i will update my working copy soon )

File: plugins.sh [hunk 1]
```diff
@@ -12,6 +12,8 @@ REF=/usr/share/jenkins/ref/plugins
 mkdir -p $REF
 
 while read spec; do
+# Skip the comments starting with #
+    [[ ${spec:0:1} == '#' ]] && continue
```
Comment (jglick) [inline plugins.sh]: Merge with `master`, this is already there.

File: plugins.txt.example [hunk 2]
```diff
@@ -0,0 +1,31 @@
+ant:1.2
+antisamy-markup-formatter:1.1
+build-flow-plugin:0.16
+credentials:1.22
+cvs:2.11
```
Comment (jglick) [inline plugins.txt.example]: I think a couple of example lines in a comment at the top of `plugins.sh` would be more helpful.

File: plugins.sh (hunk 1 above)
Comment (hayderimran7) [inline plugins.sh]: Thanks . I see this is already taken care on master branch. :+1:

File: plugins.txt.example (hunk 2 above)
Comment (hayderimran7) [inline plugins.txt.example]: Makes sense, will make a new PR for it :) thanks

**Pull request closed without merging**